### PR TITLE
Convert promisified (.then()) examples to async/await

### PR DESCRIPTION
**Pull request by inancgumus**
Note that, IMO, `examples/browser_on.js` does not need to be converted to async/await. I experimented with it, but it didn't look good. Be my guest if you have any ideas :)

Closes: #697

### CODE REVIEW

**Review by ankur22 — APPROVED** [review]
LGTM.

Happy to leave the `browser_on.js` script as is for now since those promises are being handled like pub/sub async style workflows, whereas the other tests require a more synchronous style of workflow.

File: examples/fillform.js [hunk 1]
```diff
@@ -7,36 +7,35 @@ export const options = {
...
   const context = browser.newContext();
   const page = context.newPage();
 
-  // Goto front page, find login link and click it
-  page.goto('https://test.k6.io/', { waitUntil: 'networkidle' }).then(() => {
-    return Promise.all([
+  try {
+    // Goto front page, find login link and click it
+    await page.goto('https://test.k6.io/', { waitUntil: 'networkidle' });
+    await Promise.all([
```
Comment (mdcruz) [inline examples/fillform.js:20]: @inancgumus Just for my own understanding, can we also do?
```
await page.waitForNavigation()
await page.locator('a[href="/my_messages.php"]').click()
```

**Review by mdcruz — APPROVED** [review]
LGTM!

File: examples/fillform.js (hunk 1 above)
Comment (inancgumus) [reply examples/fillform.js:20]: Yeah, it should work, but `goto` and `waitForNavigation` have different implementations beneath. Btw, `locator.click` is not yet async.

**Pull request merged**